### PR TITLE
docs: remove deleted op-sync workflow from docs

### PR DESCRIPTION
**Pull request by AJStonewee**
Hi there the [op-sync ci](https://github.com/paradigmxyz/reth/blob/main/.github/workflows/op-sync.yml) was deleted

### CODE REVIEW

File: docs/repo/ci.md [hunk 1]
```diff
@@ -39,7 +39,6 @@ The CI runs a couple of workflows:
 [integration]: https://github.com/paradigmxyz/reth/blob/main/.github/workflows/integration.yml
 [bench]: https://github.com/paradigmxyz/reth/blob/main/.github/workflows/bench.yml
 [eth-sync]: https://github.com/paradigmxyz/reth/blob/main/.github/workflows/eth-sync.yml
```
Comment (onbjerg) [inline docs/repo/ci.md:41]: lets also link to the new workflow

```suggestion
[sync]: https://github.com/paradigmxyz/reth/blob/main/.github/workflows/sync.yml
```

**Review by onbjerg — CHANGES_REQUESTED** [review]
we also need to remove them from further up in the file where it says "eth-sync" and "op-sync"

File: docs/repo/ci.md (hunk 1 above)
Comment (AJStonewee) [reply docs/repo/ci.md]: Nice catch!

**Review by onbjerg — CHANGES_REQUESTED** [review]
we still need to also remove the workflows from up top :D under `### Code` it says:

```
- **[eth-sync]**: ..
- **[op-sync]**: ..
```

this needs to be replaced with just the sync workflow

**Comment by AJStonewee** [comment]
> this needs to be replaced with just the sync workflow

done @onbjerg

**Review by onbjerg — APPROVED** [review]

**Pull request merged**